src/pseudopeople/noise.py

```diff
@@ -32,21 +32,18 @@ def noise_form(
     """
     randomness = get_randomness_stream(form, seed)
 
+    noise_configuration = configuration[form.value]
     for noise_type in NOISE_TYPES:
-        noise_configuration = configuration[form.value]
         if isinstance(noise_type, RowNoiseType):
             # Apply row noise
-            print(noise_type.name)
-            form_data = noise_type(
-                form_data, noise_configuration, randomness, noise_type.name
-            )
+            form_data = noise_type(form_data, noise_configuration, randomness)
```

Comment:
You shouldn't pass `noise_configuration` here. You want to pass the configuration specific to the row noise function, so it should be `noise_configuration[noise_type.name]`.